glean/metrics.yaml

```diff
@@ -7,8 +7,7 @@ page:
     lifetime: application
     send_in_pings:
       - page-view
-      - interaction
-      - non-interaction
```

Comment:
Please don't remove otherwise it won't show up in the existing data anymore.

Reply:
These custom pingsare not really used anywhere yet other than my own personal testing, so I think we're fine to remove them here

Reply:
Got it! Then yes, please scrap them away!